utils/args/GenericfMRISurfaceProcessingPipeline.py

```diff
@@ -3,26 +3,23 @@
 /opt/HCP-Pipelines/fMRISurface/GenericfMRISurfaceProcessingPipeline.sh
 part of the hcp-func gear
 """
-import os, os.path as op
-from .common import build_command_list, exec_command
 import logging
+import os.path as op
+
```

Comment:
Okay, so I think the issue is I've never seen `import os.path as op` - is that the standard library import? I couldn't find it in the documentation. I would recommend, `import os `or `from os.path import join` to make it explicit instead of creating an abbreviation for an import that is not standard.

Reply:
This has been an undocumented "convention" in gears that I have read since I have been at Flywheel. 

I will consider changing in the future.